diesel_compile_tests/tests/compile-fail/sqlite_insert_or_ignore_cannot_be_used_on_pg.rs

```diff
@@ -0,0 +1,23 @@
+#[macro_use] extern crate diesel;
+
+use diesel::*;
+
+table! {
+    users {
+        id -> Integer,
+    }
+}
+
+#[derive(Insertable)]
+#[table_name="users"]
+struct User {
+    id: i32,
+}
+
+fn main() {
+    let connection = PgConnection::establish("").unwrap();
+    insert_or_ignore_into(users::table)
+        .values(&User { id: 1 })
```

Comment:
Nit: It'd be more ideomatic to use `users::id.eq(1)` here (not that it matters too much for a compile-fail test